### PR TITLE
SAK-43321 Only show the profile image if we have a user id

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-43321

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5097501&bid=16768197)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Complexity increasing per file
==============================
- webcomponents/tool/src/main/frontend/grader/sakai-grader.js  1
- webcomponents/tool/src/main/frontend/grader/submission.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=5097501&bid=16768197)

**Comment by jesusmmp** [comment]
LGTM!

File: assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java [hunk 1]
```diff
@@ -497,6 +497,7 @@ public ActionReturn getGradableForSite(EntityView view , Map<String, Object> par
         data.put("submissions", submissions);
         data.put("students", students);
         data.put("groups", groups);
+        data.put("showOfficialPhoto", serverConfigurationService.getBoolean("assignment.show.official.photo", true));
```
Comment (mpellicer) [inline assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java]: Is this property already documented in default.sakai.properties? just asking, I'll approve it.

Comment (jesusmmp) [reply assignment/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java]: I'll added it here: https://github.com/sakaiproject/sakai/pull/6599/files#diff-2c1cd69e8a95ba38a890ca8eb93b8f02R1931.

**Comment by adrianfish** [comment]
@jesusmmp Thanks for the review!

**Pull request merged**